# Segments
geometry_msgs/Point[] first_points
geometry_msgs/Point[] last_points

# Circles 
geometry_msgs/Point[] centre_points
float64[] radii
